DOCS/man/input.rst

```diff
@@ -814,6 +814,20 @@ Text Manipulation
 
         This line of Lua prints "foo \\{bar}" on the OSD.
 
+``msg <level> <message> [...]``
```

Comment:
Why not extend `print-text` command with optional arguments? It can be aliased to `msg` if preferred, but both do the same thing.

Reply:
I mean the automatic concat seems pretty useful, so it's fine IMO.

Reply:
Why duplicate functionality of higher level languages? Also it's weird to provide variadic interface that doesn't do formatting and just do concant. API layer should be simplified where possible.